code/index.html

```diff
@@ -151,7 +151,7 @@ <h1>Security cameras Utrecht</h1>
             </table>
           </td>
           <td>
-            <table id="column15">
+            <table id="column35">
```

Comment:
De `id` in dit geval is juist maar wel onduidelijk waar het om gaat. Een getal wat deelbaar is door 3 en 5 is ook deelbaar door 15. Dit is de reden achter de waarde `column15`. Ik denk dat we deze voor nu graag zou willen houden. Als je kans ziet om de PR bij te werken tot alleen het bijwerken van de link kunnen we hem mergen.

Reply:
Ah dat verklaart het, cryptisch. Prima, reverted!